scripts/build-linux-docker.sh

```
@@ -187,7 +187,7 @@ echo_info "Building Ubuntu 22.04 based container for $ARCH..."
 $CONTAINER_RUNTIME build \
     --platform linux/$ARCH \
     -f docker/Dockerfile.ubuntu-22.04 \
-    -t fujisan-linux-builder:ubuntu22-$ARCH \
+    -t localhost/fujisan-linux-builder:ubuntu22-$ARCH \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)